### DISCUSSION TITLE
How to override onClick of PaneItem ?

### DISCUSSION THREAD

**Discussion by felixalguzman** [bdlukaa/fluent_ui#229] [Q&A] [answered]
I want to call a function instead of navigating to other page.

**Comment by bdlukaa** [comment, marked as answer]
You can use `PaneItemAction` instead of `PaneItem`. For more complex pane items, see:

https://github.com/bdlukaa/fluent_ui/blob/master/example/lib/main.dart#L300-L339

**Reply by felixalguzman** [reply]
Thanks
